core/node/events/stream.go

```diff
@@ -25,9 +26,15 @@ type MiniblockStream interface {
 	GetMiniblocks(ctx context.Context, fromInclusive int64, ToExclusive int64) ([]*Miniblock, bool, error)
 }
 
+type ScrubbableStream interface {
```

Comment:
Please add some documentation what it means if a stream supports this interface.

Reply:
If a stream is joinable does that always imply it is scrubbable? If that is the case it is possible to either move these 2 functions to the `JoinableStreamView` interface or extend the `JoinableStreamView` interface with this interface.

Reply:
I moved the timestamp to the view and copy it on copyAndAddEvent and copyAndApplyMiniblock. I also updated the interface name to ScrubTrackable and added some comments. PTAL!